packages/core/octo-linker.js

```diff
@@ -18,7 +18,12 @@ function initialize(self) {
 }
 
 function insertDataAttr(matches) {
-  matches.forEach(({ data, link }) => {
+  matches.forEach(item => {
```

Comment:
Another solutions would be to strip out all `undefindes` in the first place, although this would require an additional loop over `matches`. Anyway, at the end it really doesn't matter IMO